anvil/core/src/eth/mod.rs

```diff
@@ -61,6 +61,9 @@ pub enum EthRequest {
     #[serde(rename = "eth_getBlockByNumber")]
     EthGetBlockByNumber(#[serde(deserialize_with = "lenient_block_number")] BlockNumber, bool),
 
+    #[serde(rename = "eth_getBlocksByArray")]
+    EthGetBlocksByArray(#[serde(deserialize_with = "lenient_block_number_array")] Vec<BlockNumber>),
+
```

Comment:
great, let's also add the bool value so we follow `eth_getBlockByNumber`

also I'm not sure if this should be put in the `eth_` namespace since this is anvil specific call, but perhaps `eth_getBlocksByArray` is just more fitting